libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -65,7 +65,9 @@ void CodeTransform::operator()(const FunctionDefinition&)
 	solAssert(false, "Function definition not removed during desugaring phase.");
 }
 
-void CodeTransform::generateAssignment(Identifier const& _variableName, SourceLocation const& _location)
+void CodeTransform::generateAssignment(
+	Identifier const& _variableName,
+	SourceLocation const& _location)
```

Comment:
Sorry to raise this again, but if you restyle this I think the closing parentheses should be on a new line? 😉